eb_hooks.py

```diff
@@ -733,6 +734,9 @@ def post_sanitycheck_cuda(self, *args, **kwargs):
                                        basename, full_path)
                         # if it is not in the allowlist, delete the file and create a symlink to host_injections
                         host_inj_path = full_path.replace('versions', 'host_injections')
+                        # CUDA itself doesn't care about compute capability so remove this duplication from
+                        # under host_injections
+                        host_inj_path = re.sub(r"accel/nvidia/cc\d+/", '', host_inj_path)
```

Comment:
Oof, this is hardcoding the `accel/nvidia/cc*` pattern quite deep in our scripts...

Why don't we pick up on `$EESSI_ACCELERATOR_TARGET` here instead, just in case that changes?

Reply:
What's the value of `EESSI_ACCELERATOR_TARGET`?

Reply:
Found it, I'd need `accel/$EESSI_ACCELERATOR_TARGET`

Reply:
Implemented in https://github.com/EESSI/software-layer/pull/735/commits/087b7d7d42436b07c035b3433d03c9318fd9ec4c